### DISCUSSION TITLE
Stacks Range?

### DISCUSSION THREAD

**Discussion by FurtadoPires** [ReactiveDrop/reactivedrop_translations#1506] [Q&A] [answered]
I'm having difficult to translate this specific term "Stacks Range" from `RiskOfRain\Resource\ReactiveDrop` what's the meaning in game for this? Does it have a synonym in english?

I guess that "Stacks" in this case means "accumulate something"?

**Comment by FurtadoPires** [comment]
Judging by the following parameters, I guess it really means "acumulate range" so I'll be translating considering that.

**Reply by ModdedMarionette** [reply, marked as answer]
Pretty much yes.
The same type of item drops in Risk of Rain can be picked up multiple times and have their effects stacked that way. "Stacks range" basically means this item has some sort of effect that has a specific range and picking up multiple of this type of item will increase that range.
